Wireframe/style.css

```diff
@@ -50,6 +50,7 @@ main {
   margin: auto;
 }
 footer {
+  background-color: rgb(184, 134, 249);
   position: fixed;
```

Comment:
as best practice footers are generally stretched to its container, this is a design enhancement, you don't have to do it, but it is better to practice more which will improve your UI and UX skills